src/click/types.py

```diff
@@ -661,23 +662,41 @@ def _clamp(self, bound: float, dir: t.Literal[1, -1], open: bool) -> float:
 class BoolParamType(ParamType):
     name = "boolean"
 
+    bool_states: dict[str, bool] = dict(**RawConfigParser.BOOLEAN_STATES) | {
+        "t": True,
+        "f": False,
+        "y": True,
+        "n": False,
+        # Absence of value is considered False.
+        "": False,
+        # XXX Possible states to add in the future:
```

Comment:
Let's not suggest other values, because that's an endless bikeshed. Having a dict here that users can override with their own values is good enough. Although I don't think we should document that as a public API (yet).

Reply:
Agreed!